app/models/base_regulation.rb

```diff
@@ -20,6 +20,10 @@ def q_search(keyword)
         Sequel.ilike(:information_text, "#{keyword}%")
       }
     end
+
+    def fully_replaced
+      where(replacement_indicator: 1)
```

Comment:
I think this is not correct, we should be scoping it to approved regulations, not replaced ones:

Regulation approved flag = 1